QuickFlicks/Controller/MovieDetailViewController.swift

```diff
@@ -9,71 +9,141 @@
 import UIKit
 import Alamofire
 import AlamofireImage
+import SteviaLayout
 
-class MovieDetailViewController: UIViewController, UIScrollViewDelegate {
-
-    @IBOutlet weak var scrollView: UIScrollView!
+class MovieDetailView: UIView {
     
-    @IBOutlet weak var posterImageView: UIImageView!
+    var posterImageView = UIImageView()
+    var contentView = UIView()
+    var titleLabel = UILabel()
+    var dateLabel = UILabel()
+    var rateLabel = UILabel()
+    var timeLabel = UILabel()
+    var overviewLabel = UILabel()
     
-    @IBOutlet weak var titleLabel: UILabel!
+    convenience init() {
+        self.init(frame: CGRectZero)
+        
+        render()
+    }
     
-    @IBOutlet weak var dateLabel: UILabel!
+    func render() {
+        
+        backgroundColor = .whiteColor()
+        
+        // Setup UI, need to clean up later
+        // === BEGIN ===
+        posterImageView.contentMode = .ScaleAspectFill
+        
+        contentView.backgroundColor = UIColor.blackColor()
+        contentView.alpha = 0.8
+        
+        titleLabel.textColor = UIColor.whiteColor()
+        titleLabel.numberOfLines = 0
+        titleLabel.font = UIFont.boldSystemFontOfSize(17)
+        dateLabel.textColor = UIColor.whiteColor()
+        dateLabel.font = dateLabel.font.fontWithSize(13)
+        rateLabel.textColor = UIColor.whiteColor()
+        rateLabel.font = rateLabel.font.fontWithSize(13)
```

Comment:
Totally agree.
That's why I have this comment at the beginning of code block:
// Setup UI, need to clean up later
// === BEGIN ===
...